### ISSUE TITLE
Deprecation warning:

### ISSUE THREAD

**Issue by jwright04** [offirgolan/ember-cp-validations#195]
I see this was addressed here:
https://github.com/offirgolan/ember-cp-validations/pull/140

But when using the following
```
validator('date', {
after: 'now',
before: '1/1/2020',
precision: 'day',
format: 'M/D/YYY',
errorFormat: 'M/D/YYY'
})
```
I'm seeing the Deprecation warning again:

> Deprecation warning: moment construction falls back to js Date. This is discouraged and will be removed in upcoming major release. Please refer to https://github.com/moment/moment/issues/1407 for more info.
Arguments: [object Object]

Anything I can do to resolve this warning?

**Comment by offirgolan** [comment]
@jwright04 I have a PR pending with some date format issues but I wasnt able to recreate your issue. Im assuming this is because `M/D/YYY` is not a valid date format according to momentjs docs which is probably why its reverting back to js Date.

<img width="726" alt="screen shot 2016-05-25 at 1 33 20 pm" src="https://cloud.githubusercontent.com/assets/575938/15555371/51566eee-227d-11e6-9a92-786009c0386b.png">
